packages/location-management/src/components/LocationTree/tests/index.test.tsx

```diff
@@ -1,40 +1,118 @@
 
 describe('location-management/src/components/LocationTree', () => {
+  beforeEach(() => {
+    fetch.mockClear();
+  });
+
+  it('test getHierarchy', async () => {
+    fetch.mockResponseOnce(JSON.stringify(rawHierarchy[2]));
+    const response = await getHierarchy([baseLocationUnits[2]], 'accessToken', baseURL);
+    expect(response).toMatchObject([rawHierarchy[2]]);
+  });
+
+  it('test getBaseTreeNode', async () => {
+    fetch.mockResponseOnce(JSON.stringify(baseLocationUnits));
+    const response = await getBaseTreeNode('accessToken', baseURL);
+    expect(response).toMatchObject(baseLocationUnits);
+  });
+
   it('renders without crashing', async () => {
+    fetch.mockResponseOnce(JSON.stringify(baseLocationUnits));
+    fetch.mockResponseOnce(JSON.stringify(rawHierarchy[0]));
+    fetch.mockResponseOnce(JSON.stringify(rawHierarchy[1]));
+    fetch.mockResponseOnce(JSON.stringify(rawHierarchy[2]));
+
     const wrapper = mount(
       <Provider store={store}>
-        <Tree data={treedata} OnItemClick={jest.fn} />
+        <Tree opensrpBaseURL={opensrpBaseURL} accessToken={accessToken} OnItemClick={jest.fn} />
       </Provider>
     );
 
     await act(async () => {
       await flushPromises();
     });
+    wrapper.update();
+
+    expect(fetch.mock.calls).toMatchObject([
+      [
+        'https://test-example.com/rest/location/findByProperties?is_jurisdiction=true&return_geometry=false&properties_filter=status:Active,geographicLevel:0',
+        {
+          headers: {
+            accept: 'application/json',
+            authorization: 'Bearer Token',
+            'content-type': 'application/json;charset=UTF-8',
+          },
+          method: 'GET',
+        },
+      ],
+      [
+        'https://test-example.com/rest/location/hierarchy/a26ca9c8-1441-495a-83b6-bb5df7698996',
+        {
+          headers: {
+            accept: 'application/json',
+            authorization: 'Bearer Token',
+            'content-type': 'application/json;charset=UTF-8',
+          },
+          method: 'GET',
+        },
+      ],
+      [
+        'https://test-example.com/rest/location/hierarchy/b652b2f4-a95d-489b-9e28-4629746db96a',
+        {
+          headers: {
+            accept: 'application/json',
+            authorization: 'Bearer Token',
+            'content-type': 'application/json;charset=UTF-8',
+          },
+          method: 'GET',
+        },
+      ],
+      [
+        'https://test-example.com/rest/location/hierarchy/6bf9c085-350b-4bb2-990f-80dc2caafb33',
+        {
+          headers: {
+            accept: 'application/json',
+            authorization: 'Bearer Token',
+            'content-type': 'application/json;charset=UTF-8',
+          },
+          method: 'GET',
+        },
+      ],
+    ]);
 
     expect(wrapper.find('.ant-tree')).toHaveLength(1);
   });
 
   it('test tree search functionality', async () => {
+    // fetch.mockResponseOnce(JSON.stringify(baseLocationUnits));
```

Comment:
Let's remove this commented lines of code

Reply:
done